CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+# CMake configuration for ECOS
+
+cmake_minimum_required(VERSION 3.5)
+
+project(ecos
+        VERSION 2.0.7
+        LANGUAGES C)
+
+# Options
+option(USE_LONG "Whether to use Long or Int for index type." ON)
+
+# Set compiler flags
+set(DEBUG_OPTIONS -O0 -Wall -Wextra)
+set(RELEASE_OPTIONS -O3)
+
```

Comment:
These won't work like this on Windows - we can either surround this with a check to only happen on Linux and Mac (both worked when I tested this) or at least add a comment somewhere that says this currently does not support Windows builds.